meta-elbb-raspberrypi/recipes-bsp/rpi-uboot-scr/rpi-u-boot-scr/boot.cmd.in

```diff
@@ -0,0 +1,7 @@
+saveenv
+fdt addr ${fdt_addr} && fdt get value bootargs /chosen bootargs
+if env exists rpipart;then echo Booting from mmcblk0p${rpipart};else setenv rpipart 2;echo rpipart not set, default to ${rpipart};fi
+#load mmc 0:${rpipart} ${kernel_addr_r} boot/@@KERNEL_IMAGETYPE@@
```

Comment:
should it removed that commented code part?

Reply:
nope, actually it would be the correct version to load a kernel from either A or B partition. The problem is currently the kernel is installed is installed in neither A nor B.
As soon as we fix it this will be the correct line and the next line gets deleted.